src/Database/Driver.php

```diff
@@ -32,7 +32,7 @@ abstract class Driver implements DriverInterface
     /**
      * Instance of PDO.
      *
-     * @var \PDO|null
+     * @var \PDO
```

Comment:
While this property is not set in constructor I chose to remove the `|null` to avoid having to replace all `$this->_connection` usage with `$this->getConnection()` calls in all drivers. With numerous usage of PDO connection instance during a request cycle the methods calls (which have no null) checks could end up adding a non trivial amount of overhead.